core/src/test/kotlin/io/fritz2/binding/seq.kt

```diff
@@ -80,39 +80,39 @@ class SeqTests {
 
                 button("replaceList") {
                     +"replaceList"
-                    store.replaceList <= clicks
+                    store.replaceList <= clicks()
                 }
                 button("addAtBeginning") {
                     +"addAtBeginning"
-                    store.addAtBeginning <= clicks
+                    store.addAtBeginning <= clicks()
```

Comment:
Not sure if invoke is a good idea. It looks good when you use it, but will it be clear that it is `clicks()` but `clicks.map {...}`?